Simplenote/Information/NoteInformationViewController.swift

```diff
@@ -177,6 +181,24 @@ private extension NoteInformationViewController {
 
         additionalSafeAreaInsets = UIEdgeInsets(top: headerStackView.frame.maxY, left: 0, bottom: 0, right: 0)
     }
+    
+    func configureDragBar() {
+        let dragBar = UIView(frame: CGRect(x: 0, y: 0, width: 36, height: 5))
+        dragBar.backgroundColor = UIColor.black
+        dragBar.alpha = 0.2
+        dragBar.layer.cornerRadius = 2.5
+        dragBar.layer.masksToBounds = true
+        
+        view.addSubview(dragBar)
+        
+        dragBar.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            dragBar.topAnchor.constraint(equalTo: view.topAnchor, constant: 5),
+            dragBar.bottomAnchor.constraint(equalTo: view.topAnchor, constant: 10),
```

Comment:
I think it's better to use `height`. Easier to understand dimensions :-)

Reply:
I have made those changes.  Check it out!  Let me know if you see anything else that needs to be updated!